app/assets/stylesheets/partials/_footer.scss

```diff
@@ -1,28 +1,23 @@
 .footer {
+  @include media(medium) {
```

Comment:
This makes a lot more sense to me than having a `footer` class largely styled by its child `container` class. Nice!